prql-net/prql-net.sln

```diff
@@ -0,0 +1,28 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PrqlCompiler", "PrqlCompiler\PrqlCompiler.csproj", "{339EA2A6-23D2-4938-884F-052431AC0674}"

```

Comment:
Can I ask where this comes from?

Does it need to be updated in the future? Would it be possible to write a couple of sentences on this in a comment or the Readme?

Reply:
No, it doesn't need to be updated. That gets auto-generated when you create a new solution (`.sln`) file using `dotnet new sln` and add the projects to the solution using `dotnet add reference PrqlCompiler\PrqlCompiler.csproj`.

Reply:
Technically the solution file isn't really needed, its just a convenient thing so that you if you use Visual Studio then you can open the solution file which holds both the compiler project and the unit test project. It is possible for `dotnet` to compile the '.csproj` project files, so we could remove the solution file if you don't like it.